community_tasks/swiss_legal_evals.py

```diff
@@ -0,0 +1,402 @@
+# MIT License
+
+# Copyright (c) 2024 The HuggingFace Team
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# ruff: noqa: F405, F403, F401
+"""
+This module contains task configurations and prompt functions for evaluating
+LLM models on Swiss legal datasets. Each task is defined using the 
+`LightevalTaskConfig` class with its respective prompt function. The tasks 
+cover a variety of benchmarks, including: translation of laws, court decisions 
+and press releases.
+
+Author: Joel Niklaus
+"""
+from lighteval.logging.hierarchical_logger import hlog_warn
+import nltk
+
+nltk.download("punkt_tab")
+
+
+import numpy as np
```

Comment:
Numpy is not a core dep of lighteval, this is likely to fail - do you really need to use numpy or can you use something else?